# This message describes the O-79 imaging radar device parameters.

string fw_version     # Firmware version
string can_id         # Radar CAN ID
string serial_no      # Radar serial number
string mac_address    # Radar MAC address